### PR TITLE
Finished List Lab Exercise

### PR DESCRIPTION
**Pull request by shaynaalice**

### CODE REVIEW

**Comment by shaynaalice** [comment]
Can you let me know If I'm doing the if __name__ == '__main__': correctly? For some reason, this piece really confuses me. Thanks!

File: students/shayna_andersonhill/lesson03/list_lab.py [hunk 1]
```diff
@@ -16,9 +17,54 @@
...
+print(fruits)
+response = input("Please user, give me a fruit to delete > ")
+fruits.remove(response)
+print(fruits)
+
+#Series 3
+response_list = []
+for i in fruits:
+    response = input('Do you like {}? '.format(i.lower()))
+    while response != 'yes' and response != 'no':
```
Comment (mdgregor) [inline students/shayna_andersonhill/lesson03/list_lab.py]: `while response not in ('yes', 'no'):`

File: students/shayna_andersonhill/lesson03/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    print('------------------------------------------------------------------')
+    for donor, total, num, average in zip(donors, total_given, num_gifts,
+        average_gifts):
+        print('{:20}    ${:12.2f}{:>12}   ${:>12.2f}'.format(donor, total, num, average))
+
+
+
+
+
+if __name__ == '__main__':
```
Comment (mdgregor) [inline students/shayna_andersonhill/lesson03/mailroom.py]: yep this is fine.  but typically you would define a `main()` function and put all the stuff under this in that.

File: students/shayna_andersonhill/lesson03/mailroom.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    response = input('Would you like to "Send a Thank You", "Create a Report", or "quit"?')
+    name_list = [i[0] for i in donations]
+
+    donors = []
+    for i,j in donations:
```
Comment (mdgregor) [inline students/shayna_andersonhill/lesson03/mailroom.py]: `i` and `j` don't tell me anything about what you are iterating threw,  use something like `for name, donation in donations`

**Pull request merged**